Docker.DotNet/Endpoints/IMiscellaneousOperations.cs

```diff
@@ -21,8 +21,14 @@ public interface IMiscellaneousOperations
 
         Task<Stream> GetImageAsTarballAsync(string name, CancellationToken cancellationToken);
 
+        Task<Stream> GetMultipleImagesAsTarballAsync(string[] names, CancellationToken cancellationToken);
+
         Task LoadImageFromTarballAsync(Stream stream, CancellationToken cancellationToken);
 
+        Task<Stream> LoadImageFromTarballWithProgressAsync(Stream stream, CancellationToken cancellationToken);
```

Comment:
A side note. If we moved `Tar` from Docker PowerShell back to its own then we could have Docker.DotNet reference it too and we could take methods like this as `(TarStream stream, ...)`.

What do you think?